Document limitations of namespace-scoped KUDO resources (#247)
kudobuilder/www: content/docs/what-is-kudo.md

```bash
$ kubectl kudo install kafka --instance dev-kafka
```

### Limitations

> The _Operator_, _OperatorVersion_, and _Instance_ resources created by KUDO are namespace scoped. These resources can only own resources in the same namespace. As a result, it isn't possible for an operator to create resources in multiple namespaces.

Reviewer: ```suggestion
The _Operator_, _OperatorVersion_, and _Instance_ resources created by KUDO are namespace scoped. These resources can only own resources in the same namespace. As a result, it isn't possible for a single operator to create resources in multiple namespaces.
```

> The _Operator_, _OperatorVersion_, and _Instance_ resources created by KUDO are namespace scoped. These resources can only own resources in the same namespace. As a result, it isn't possible for a single operator to create resources in multiple namespaces.

Reviewer 2: ```suggestion
All the resources created by an operator _Instance_ are [owned](https://kubernetes.io/docs/concepts/workloads/controllers/garbage-collection/#owners-and-dependents) by it. This way, when the _Instance_ is deleted, the resources are removed too. However, the _Instance_ (as well as  _Operator_, and _OperatorVersion_)  are namespace scoped. Namespaced resources can only own resources in the same namespace. As a result, it isn't possible for a single operator to create resources in multiple namespaces.
```

> It is possible for an _Instance_ to create cluster-scoped resources. However, this can result in issues when updating or upgrading an _Instance_. [KEP-5](https://github.com/kudobuilder/kudo/blob/master/keps/0005-cluster-resources-for-crds.md) will resolve this limitation.

Reviewer 2: ```suggestion
It is possible for an _Instance_ to create cluster-scoped resources. However, these `resourceOwners` field for such resources is not populated with the _Instance_ reference. Cluster-scoped resources are also **not** automatically cleaned up when the _Instance_ is deleted. Additionally,  they can result in issues when updating or upgrading an _Instance_. [KEP-5](https://github.com/kudobuilder/kudo/blob/master/keps/0005-cluster-resources-for-crds.md) will resolve this limitation.
```
